library/kotlin/src/io/envoyproxy/envoymobile/stats/Counter.kt

```diff
@@ -7,5 +7,6 @@ interface Counter {
   /**
    * Increment the counter by the given count.
    */
+  // TODO: return A status indicating if the action was successful.
```

Comment:
nit: I'd drop this TODO for now - I'm not certain we _do_ want to expose this publicly via a return code. (We can also throw an exception. Or log. Or do nothing.)